boot/bootutil/src/image_ed25519.c

```diff
@@ -9,6 +9,11 @@
 
 #include "mcuboot_config/mcuboot_config.h"
 
+#if defined(CONFIG_NRF_SECURITY)
+/* We are not really using the MBEDTLS but need the ASN.1 parsing funcitons */
```

Comment:
Minor detail, but I don't agree much with this comment. You are not using Mbed TLS for crypto, but you are using its ASN1 parse capabilities. So I would rephrase it as:

/* Enable ASN1 parse functions from Mbed TLS */

Wdyt?

Reply:
Yeah, I can replace that. btw:https://github.com/nrfconnect/sdk-mcuboot/pull/326